packages/application-config/schema.json

```diff
@@ -30,7 +30,7 @@
       "oneOf": [
         {
           "type": "string",
-          "pattern": "^[^\\-_]([0-9a-z]|[\\-_](?![\\-_])){2,64}[^\\-_]$"
+          "pattern": "^[^\\-_#]([0-9a-z]|[\\-_](?![\\-_])){0,62}[^\\-_#]$"
```

Comment:
I assume that since # is kind of like a special url character, maybe we want to check in the character class `[^-_#]` beginning and end to be sure the entryPathUriPath does not start or end with it too.
Any thoughts on this?

Reply:
Good idea, thanks for noticing.

Let's add it to the regex as you suggested, thanks!